ccm_web/server/src/api/api.utils.ts

```diff
@@ -28,16 +36,27 @@ export function handleAPIError (error: unknown, input?: string): APIErrorPayload
     return { canvasStatusCode: statusCode, message: bodyText, failedInput: failedInput }
   } else {
     logger.error(`An error occurred while making a request to Canvas: ${JSON.stringify(error)}`)
-    return { canvasStatusCode: 500, message: 'A non-HTTP error occurred while communicating with Canvas.', failedInput: failedInput }
+    const defaultMessage = 'A non-HTTP error occurred while communicating with Canvas.'
+    return { canvasStatusCode: HttpStatus.INTERNAL_SERVER_ERROR, message: defaultMessage, failedInput: failedInput }
   }
 }
 
 export function parseErrorBody (body: unknown): string {
   if (body === null || body === undefined || String(body).startsWith('<!DOCTYPE html>')) return 'No response body was found.'
   if (!isCanvasErrorBody(body)) {
+    logger.debug('Supposedly: Response body had unexpected shape')
```

Comment:
The debug statement wasn't meant to be snark.  That was me thinking out loud in the code.  As in, "supposedly there's a problem here, but I don't understand what it is or why".

It's obvious.  I mean, you'd know that if you weren't so…

Oh, never mind.

😆

Reply:
SRSLY tho, those logger debug statements aren't really necessary any longer.

Reply:
🏁

Reply:
Sorry if I was a little rough on you here. When I'm reviewing something, I mostly think you're proposing changes that will get merged. If you wanted something with "supposedly" merged, I'd consider that snarky. Anyways, I was just trying to say let's address the root problem or improve the message generation, rather than add more debug messages.